### PR TITLE
[#319] Add support for thresholds of aggregated metrics

### LINKED ISSUE #319: Add configurable thresholds to report aggregator
As implemented in https://github.com/hcoles/pitest/pull/1095 . PIT 1.9.9 (most likely - currently in master).

### PR DESCRIPTION
**Pull request by Pfoerd**
Fixes #319

### CODE REVIEW

**Comment by C-Otto** [comment]
If you use "fixes" instead of "solves", GitHub will automatically link the issue (and close it once this PR is merged).

**Review by szpak — APPROVED** [review]
Looks good. Thanks. I plan to just do some minor tweaks and merge it in the next "merge window" (hopefully this week).

File: src/funcTest/resources/testProjects/multiproject/build-report.gradle [hunk 1]
```diff
@@ -42,6 +42,11 @@ subprojects {
             outputFormats = ["HTML","XML"]
             timestampedReports = false
             exportLineCoverage = true
+            reportAggregator {
+                aggregatedTestStrengthThreshold.set(50)
+                aggregatedMutationThreshold.set(40)
+                aggregatedMaxSurviving.set(3)
```
Comment (szpak) [inline src/funcTest/resources/testProjects/multiproject/build-report.gradle:48]: @Pfoerd I sat down to merge it and re-reading the changes, I wonder, if they should be named without the `aggregated` prefix (as they are in the `reportAggregator` parent)?

```
reportAggregator {
    testStrengthThreshold.set(50)
    mutationThreshold.set(40)
    maxSurviving.set(3)
}
```

(just asking about your opinion - no need to perform any work on your side - potential changes I would do on my own after the merge)

Comment (Pfoerd) [reply src/funcTest/resources/testProjects/multiproject/build-report.gradle:48]: Looks good, I agree that the prefixes are actually not needed when using the nesting syntax) 👍

Comment (szpak) [reply src/funcTest/resources/testProjects/multiproject/build-report.gradle:48]: Btw, I've forgotten that the groovy syntax sugar testStrengthThreshold = 50`doesn't work for nested properties :-).

**Pull request merged**